goto/tests/test_end_to_end.sh

```diff
@@ -116,7 +127,7 @@ function _projectfile_should_contain {
 
 function _projectfile_should_not_contain {
     local magicword="$1"
-    if [[ $(cat $PROJECTFILE | grep $magicword | wc -l) -ne 0 ]]; then
+    if [[ $(cat $PROJECTFILE | grep "\"$magicword\"" | wc -l) -ne 0 ]]; then
```

Comment:
this fixes a bug in the test framework itself -- it now searches for magicwords with the `"` quotes around it.
This is to make sure that grep is not returning multiple results for two magicwords with a common base string:

`test_1` and `test_123` for instance